libs/models/src/lib/sockets/player.ts

```diff
@@ -25,18 +31,19 @@ export enum PlayerState {
   Racing = 'racing',
 }
 
+// ==== Types ==== //
+/** Player id template literal */
+export type playerId = z.input<typeof playerIdSchema>;
```

Comment:
Can we have this renamed to `PlayerId`

Reply:
fixed de1f40717ab164f66408e3bf2acbe282ff4ce7e7